server/src/HatoholArmPluginGateHAPI2.cc

```diff
@@ -321,6 +322,104 @@ struct HatoholArmPluginGateHAPI2::Impl
 			MLPL_WARN("Failed to call %s\n", m_methodName.c_str());
 		}
 	};
+
+	bool isPluginControlScriptAvailable(void)
+	{
+		if (m_pluginInfo.path.empty())
+			return false;
+		bool executable = g_file_test(m_pluginInfo.path.c_str(),
+					      G_FILE_TEST_IS_EXECUTABLE);
+		if (!executable)
+			MLPL_DBG("The plugin control script isn't executable:"
```

Comment:
> Or, is a message of failure of the execution shown in other place ?

Yes, the failure message is shown in runPluginControlScript. However, the reason isn't.
So here the message including a cause should be logged with more higher level.